bika/lims/browser/bika_listing.py

```diff
@@ -661,7 +661,7 @@ def _process_request(self):
                 continue
             request_key = "%s_%s" % (form_id, index)
             value = self.request.get(request_key, '')
-            if len(value) > 1:
+            if len(value) > 0:
```

Comment:
Not sure about this change, but this condition is also used in https://github.com/senaite/bika.lims/pull/480/files#diff-05737f388606e64fa9543e7eea467c36R680 . So, if this changes here, I guess this change must be applied in Line 680 too. Maybe there are other places where this condition is used?

If I understand correctly, with `len(value) > 1` we prevent the system to do searches if the search term does not have at least 2 characters? Can you confirm @Espurna

Reply:
Indeed, think about a case where we have '1' and '0' as True/False indicators. In my opinion this conditional should check whether there is data.

Reply:
Good point